File: compare_languages/cpp/stats.cpp
#include <iostream>
#include <vector>

int main() {
    std::vector<int> numbers = {1,4,6,8,10};

    int total = 0;
    for (int x : numbers) {
        total += x;
    }

    double mean = static_cast<double>(total) / numbers.size();

    int countAboveMean = 0;
    for (int x : numbers) {
        if (x > mean) {
            countAboveMean++;
        }
    }

    std::cout << "Sum: " << total << std::endl;
    std::cout << "Mean: " << mean << std::endl;
    std::cout << "Count above mean: " << countAboveMean << std::endl;

    return 0;
}

// run code: g++ cpp/stats.cpp -o cpp/stats
// run code: stats.exe

File: compare_languages/python/stats.py
numbers = [1,4,6,8,10]

total = sum(numbers)
mean = total / len(numbers)

count_above_mean = sum(1 for x in numbers if x > mean)

print(f"Sum: {total}")
print(f"Mean: {mean}")
print(f"Count above mean: {count_above_mean}")

# Run: python python/stats.py